webpack/components/ProxmoxServer/ProxmoxServerHardware.js

```diff
@@ -0,0 +1,204 @@
+import React, { useState } from 'react';
+import { Button, Title, Divider, PageSection } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import PropTypes from 'prop-types';
+import InputField from '../common/FormInputs';
+import ProxmoxComputeSelectors from '../ProxmoxComputeSelectors';
+import CPUFlagsModal from './components/CPUFlagsModal';
+
+const cpuFlagNames = [
+  'md_clear',
+  'pcid',
+  'spec_ctrl',
+  'ssbd',
+  'ibpb',
+  'virt_ssbd',
+  'amd_ssbd',
+  'amd_no_ssb',
+  'pdpe1gb',
+  'hv_tlbflush',
+  'hv_evmcs',
+  'aes',
+];
+
+const cpuFlagDescriptions = {
```

Comment:
Knit-pick: for some reason, the order in the UI differs from the order in the proxmox interface. But this is really, really minor.